python/pyiceberg/schema.py

```diff
@@ -890,6 +892,22 @@ def __init__(self) -> None:
         self._field_names: List[str] = []
         self._short_field_names: List[str] = []
 
+    def before_map_key(self, key: NestedField) -> None:
+        self.before_field(key)
```

Comment:
Isn't this the default implementation?

Reply:
Yes, looks like it. Let me clean that up